vowpalwabbit/parse_example_json.h

```diff
@@ -1708,15 +1714,15 @@ void read_line_decision_service_json(vw& all, v_array<example*>& examples, char*
 
   apply_pdrop(all, data->probabilityOfDrop, examples);
 
-  if (!result.IsError()) return;
-
-  BaseState<audit>* current_state = handler.current_state();
+  if (result.IsError())
+  {
+    BaseState<audit>* current_state = handler.current_state();
+    logger::errlog_warn("JSON parser error at {0}: {1}. Handler: {2} State: {3}", result.Offset(),
+        GetParseError_En(result.Code()), handler.error().str(), (current_state ? current_state->name : "null"));
```

Comment:
Per discussion in other PR, perhaps we should only log this a set number of times, before then just not logging for the rest of this input. e.g. log 10 times and then say "there are a lot of parse errors in this file". The metric will still keep count of how many errors (which is super nice :) )

Reply:
I think VW should not be that opinionated about silencing selectively, i.e. library mode or other bindings want to know that information all the time. Let's think about it for a bit.

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/pull/3021/files

Reply:
#3021